frontend/web/components/modals/CreateFlag.js

```diff
@@ -187,6 +198,50 @@ const CreateFlag = class extends Component {
     ) {
       this.getFeatureUsage()
     }
+
+    if (
```

Comment:
Could we put all of this into a separate components (e.g. the github integration, github issue select(allow a prop to include prs)? CreateFlag is already very bloated. Also that way the hooks can be used to cut down the amount of code this is calling

Reply:
Ok, I created some new components.